apps/backend/src/routes.ts

```diff
@@ -69,24 +69,27 @@ export function registerRoutes(app: Express, deps: Dependencies) {
   app.get("/api/workspaces/:workspaceId/members", authMiddleware, workspace.getMembers)
 
   // ===========================================================================
-  // Stream routes
+  // Stream routes (all workspace-scoped)
```

Comment:
While not really part of this PR, can we please remove these? We already get the same benefit using visually separating newlines. No need for these block style comments.